docs/_sass/_search-bar.scss

```diff
@@ -4,7 +4,10 @@
 
 $color-appBG: $color-green-appBG;
 $color-highlightBG: $color-green-highlightBG;
+$color-highlightBG-hover: $color-green-highlightBG-hover;
 $color-accent : $color-green400;
+$color-button-background: $color-button-background;
+$color-button-hovered: $color-button-hovered;
```

Comment:
Where is this getting it's value from?

Reply:
colors.scss

Reply:
Sure thing, the number 2 is hovered
<img width="377" alt="Screenshot 2023-08-16 at 23 06 08" src="https://github.com/Expensify/App/assets/20473526/4f338fdd-b92b-4794-a920-6901be6d4645">

Reply:
Maybe you need to push some code up? I guess what I am saying is, I don't see how the var `$color-button-hovered` has an actual value, because you are giving it a value of itself (the same var name).

Reply:
Ahh yess, It's pushed now @shawnborton